commands/files/serialfile.go

```diff
@@ -84,10 +84,25 @@ func (f *serialFile) NextFile() (File, error) {
 	// open the next file
 	fileName := fp.Join(f.name, stat.Name())
 	filePath := fp.Join(f.path, stat.Name())
+	st, err := os.Lstat(filePath)
```

Comment:
i.e. the dereference flag will appear whenever there is a recursive flag?
What will be the timescale of this PR?

Considering this to be backward incompatible (always dereference -> maybe dereference), for now it is faster to make a refactor os.Stat -> os.Lstat with the same behavior (i.e. always dereference) except that it detects broken symlinks.